blogpost.html

```diff
@@ -1,162 +0,0 @@
-<!DOCTYPE html>
-<html>
-
-<head>
-    <title>Blog Post</title>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
-    <link rel="stylesheet" href="blogpost.css" />
-</head>
-
-<body>
-
-    <main>
-
-        <section id="header">
-            <div class="title">
-                <a id="postonly" href="homepage.html">POST ONLY</a>
```

Comment:
This link is broken because there is no `homepage.html` file. If you name your homepage `index.html` (which you should always do) you can link to `"/"` and it will automatically route you to `index.html`.